homeassistant/components/plex/strings.json

```diff
@@ -39,7 +39,7 @@
     },
     "abort": {
       "all_configured": "All linked servers already configured",
-      "already_configured": "This Plex server is already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
```

Comment:
We could lost some information by using the common string. The error is not about a service already configured but the integration of one specific Plex server among more than one associated to user Plex account. Should ask to the integration owner

Reply:
That's correct. Each server is configured independently, even if they're linked to the same account. I think "device" is more fitting here.

Reply:
I did this because `already_configured_device` is used for integrations that use physical devices. Plex is a (software) service running on a device. 
There is also a common string for duplicate integrations for software and hardware integrations `single_instance_allowed`

> eddit:
> `Pi-hole` uses service and `kodi`, `forked-daapd` and `Volumio` uses device.